### PR TITLE
Add supporting of custom fdl domain

### PR DESCRIPTION
**Pull request by renkelvin**
Add supporting of custom fdl domain

### CODE REVIEW

File: Firebase/Auth/Source/FIRAuthErrorUtils.m [hunk 1]
```diff
@@ -407,6 +407,13 @@
 static NSString *const kFIRAuthErrorMessageNullUser = @"A null user object was provided as the "
     "argument for an operation which requires a non-null user object.";
 
+/** @var kFIRAuthErrorMessageInvalidDynamicLinkDomain
+    @brief Message for @c kFIRAuthErrorMessageInvalidDynamicLinkDomain error code.
+ */
+static NSString *const kFIRAuthErrorMessageInvalidDynamicLinkDomain = @"The"
+    "Firebase Dynamic Link domain used is either not configured or is unauthorized"
```
Comment (paulb777) [inline Firebase/Auth/Source/FIRAuthErrorUtils.m]: Missing space between concatenated strings.

Comment (renkelvin) [reply Firebase/Auth/Source/FIRAuthErrorUtils.m]: Nice catch, thanks!

**Comment by renkelvin** [comment]
Updated with a tiny change.
Please take a another look and I'm going to merge it. Thank you!

**Comment by paulb777** [comment]
LGTM

**Pull request merged**